### PR TITLE
Return artist instead of performer for vorbis tags

### LINKED ISSUE #89: Why does metadataVorbis#Composer and #Artist return a value that was unasked for?
I encountered this issue with my program [alto](https://github.com/ItsLychee/alto) where I was expecting the `%artist%` variable to return `m.c["artist"]`, which is the actual `artist` field defined within the metadata, but it returned `m.c["performer"]` instead. Unbeknownst to me, I thought this was an error with my code, so after a few minutes of trial and error still yielded the same result. I was puzzled, but then decided to check Windows Explorer and said that the artist field was defined. I was confused at why it wasn't returning `m.c["artist"]` when Windows Explorer was, but that was when I finally decided to conclude my confusion by going directly to the source only to see the following code:

https://github.com/dhowden/tag/blob/master/vorbis.go#L175-L185
```golang
func (m *metadataVorbis) Artist() string {
	// PERFORMER
	// The artist(s) who performed the work. In classical music this would be the
	// conductor, orchestra, soloists. In an audio book it would be the actor who
	// did the reading. In popular music this is typically the same as the ARTIST
	// and is omitted.
	if m.c["performer"] != "" {
		return m.c["performer"]
	}
	return m.c["artist"]
}
```
It turned out that `m.c["performer"]` was `" "` which caused the library to return it instead.

ID3v1 and ID3v2 doesn't do this, so why does Vorbis? Metadata for each audio file varies, and when I call a method like `Artist`, I expect `m.c["artist"]`, not `m.c["performer"]`.

I was originally thinking about using `Metadata#Raw()` to mitigate this, but after seeing the note that says that it is an unstable mapping, which was expected, I couldn't risk instability for my program.

### PR DESCRIPTION
**Pull request by melvyn2**
Also fix possibly a mistake where the empty performer tag would be returned for the composer method.

Fixes #89

### CODE REVIEW

**Comment by itslychee** [comment]
Thanks for the PR! I should have done this after filing an issue, such behavior shouldn't exist in tag.

**Comment by bertvandepoel** [comment]
@dhowden have you considered this PR already? It seems quite sensible to me

File: vorbis.go [hunk 1]
```diff
@@ -195,15 +191,11 @@ func (m *metadataVorbis) AlbumArtist() string {
 }
 
 func (m *metadataVorbis) Composer() string {
-	// ARTIST
```
Comment (dhowden) [inline vorbis.go:198]: Include the PERFORMER blurb here so that we can keep track.

**Review by dhowden — CHANGES_REQUESTED** [review]
So this is interesting.  I suspect I set this up to make the classical "more correct", but I don't actually have any files using this code so don't see why we shouldn't change it!

**Pull request merged**